### PR TITLE
Add Emergency banner

### LINKED ISSUE #395: Add the Emergency banner pattern in a new section under the 'Banner' page in Patterns.
### What does it do?
- It communicates emergency information and provides advice on what to do.
- When an emergency is no longer relevant, the content will be archived or removed.

### When to use it?
- When you need to communicate information that is urgent or critical, which will potentially affect a large group of our users. For example, if there is an emergency situation announced in Birmingham, the banner would advise all users on travel updates and guidance.
- The emergency banner should be placed directly below the main navigation.
- Only show one emergency banner notification at a time.

### When not to use it?
- When a warning or emergency is directly related to a task the user is completing. In this case, use the appropriate [Message](https://designsystem.wmnetwork.co.uk/components/message/) component to notify users of any issues.
- If the issue is related to a service we are providing such as our customer service, you should place the [Warning text](https://designsystem.wmnetwork.co.uk/components/warning-text/) and [Inset text](https://designsystem.wmnetwork.co.uk/components/inset-text/) components next to the affected service, detailing the issue.
- Any disruptions to travel will be highlighted on the home page through the [Disruption indicator](https://designsystem.wmnetwork.co.uk/components/disruption-indicators/) components.

### How it works
- The user can dismiss the Emergency banner through the 'Close' link. This will remove the current emergency banner from all pages they visit on the website.
- The On-page variant is only shown when a user is on the Emergency advice page, which can be accessed through the 'Read our latest advice' link.

### Sitewide

> Insert Sitewide Emergency banner

### On-page variant

> Insert On-page Emergency banner

### PR DESCRIPTION
**Pull request by gldgrnt**
I'm not 100% convinced this is the best way to do this, so let me know if you know of a better way. Fixes #395

### CODE REVIEW

File: src/wmnds/assets/sass/_vars.scss [hunk 1]
```diff
@@ -18,7 +18,8 @@ $palettes: (
   information: #84329b,
   disable: #676869,
   plannedDisruption: #ffdd00,
-  background: #f3f2f1
+  background: #f3f2f1,
+  emergency: #231f20
```
Comment (daylesalmon) [inline src/wmnds/assets/sass/_vars.scss:22]: I think this is the same colour as the "text" var.
Just update to use the text colour instead and remove this one.

File: src/wmnds/patterns/banner/_banner.scss [hunk 2]
```diff
@@ -3,6 +3,20 @@
   padding: $size-md 0;
   border-bottom: 1px solid get-color(text);
 
+  &--emergency {
+    padding: $size-md;
+    border-bottom: solid 3px get-color(error);
+    color: $white;
+    background-color: get-color(emergency);
```
Comment (daylesalmon) [inline src/wmnds/patterns/banner/_banner.scss:10]: Change background colour to "text" as mentioned earlier.

**Review by daylesalmon — CHANGES_REQUESTED** [review]
A few minor changes and it's ready to go.

**Review by houbly — APPROVED** [review]

**Review by catiarodriguescosta — APPROVED** [review]
Looks pretty good to me!

**Review by houbly — APPROVED** [review]

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**